src/Sulu/Bundle/MediaBundle/Controller/MediaStreamController.php

```diff
@@ -116,13 +118,31 @@ protected function getFileResponse(
         $fileSize = $fileVersion->getSize();
         $storageOptions = $fileVersion->getStorageOptions();
         $mimeType = $fileVersion->getMimeType();
-        $version = $fileVersion->getVersion();
+        $storageName = $fileVersion->getStorageName();
+        $pathInfo = pathinfo($fileName);
 
-        $path = $this->getStorage()->load($fileName, $version, $storageOptions);
+        $resourcePath = $this->getStorageManager()->load($storageOptions, $storageName);
```

Comment:
It seems to me that the `$resourcePath` is defined to early. If the `StorageManager` returns a download URL the resource path isn't used?